tests/e2e/e2e_common.go

```diff
@@ -33,6 +33,7 @@ const (
 	adminUser                                  = "Administrator@vsphere.local"
 	apiServerIPs                               = "API_SERVER_IPS"
 	attacherContainerName                      = "csi-attacher"
+	audienceForSvcAccountName                  = "https://kubernetes.default.svc.cluster.local"
```

Comment:
It is good if you could group these 5 new env variable  with some valid comment 
 like //Used for dev-opes persona

Reply:
Done